integrations/zendesk/src/definitions/index.ts

```diff
@@ -0,0 +1,44 @@
+import type { IntegrationDefinitionProps } from '@botpress/sdk'
+import { z } from 'zod'
+
+export { actions } from './actions'
+export { events } from './events'
+export { channels } from './channels'
+
+export const configuration = {
+  schema: z.object({
+    baseURL: z.string({
```

Comment:
**nitpick:** why not `organization` or `organizationDomain` or `subdomain` ?

This make it seams as if it was a locally running server or somthing